test/vm.install

```diff
@@ -2,6 +2,9 @@
 # image-customize script to install cockpit/ws container and the locally built RPM
 set -eux
 
+# HACK: https://github.com/projectatomic/atomic/issues/1258
+export ATOMIC_VAR_LIB=/var/lib/containers/atomic
```

Comment:
Let's guard this with `if [ "$1" = fedora-atomic ]`, as {rhel,continuous}-atomic don't have that bug and get broken by this.

Reply:
this script is called without any arguments so `$1` is not defined. `mkdir -p /var/lib/atomic` had to do.